### ISSUE TITLE
Counter on PB showing trackers on DDG and now GitHub

### ISSUE THREAD

**Issue by Maxxxxxx** [EFForg/privacybadgerfirefox#307]
I've been working with it better since I got some advice.
Today, I noticed the counter, which I must have had set at "off" and just turned on.    
I use DDG for searches and on one page (I have a couple of tabs open simultaneously), the counter is showing nine trackers.  On the next tab, it's showing nine.  On the third tab, it cleared to green and -0-.
Now, as I entered GH, the counter is still at 9, HERE.

I don't think I"m being tracked, but PB is telling me otherwise.

Thank you. 
![privacy badger](https://cloud.githubusercontent.com/assets/10837831/6422869/ac67fe38-be92-11e4-837e-c199b10b02a6.jpg)

**Comment by cooperq** [comment]
Can you give me a screenshot after you click on the privacy badger Icon so I can see the list of trackers? This definitely seems like a bug, but I can't reproduce it so far. Also what other addons are you running?

**Comment by cooperq** [comment]
Okay I have looked into this pretty deeply. I suspect that the cause of this was `settingsMap` being initialized to the same tab object for all the different tabs, and not changing. This would cause the tabs to not update and therefore you would see trackers on duckduckgo. 
I have not been able to figure out what would cause this, nor have I been able to reproduce it. I have looked at the code paths surrounding this logic though and it looks pretty tight. My suspicion right now is that there was an uncaught exception in privacy badger or a bug in firefox that we were hitting, which has now been fixed. 
@Maxxxxxx are you still having this issue?

I am going to close this for now unless it comes up again.
